run-mlreco/install_mlreco.sh

```diff
@@ -91,8 +91,8 @@ cd ..
 # commit 8103996
 # git clone -b jw_dune_nd_lar https://github.com/chenel/lartpc_mlreco3d.git
 
-git clone -b v2.9.5 https://github.com/DeepLearnPhysics/lartpc_mlreco3d.git
-
+#git clone -b v2.9.5 https://github.com/DeepLearnPhysics/lartpc_mlreco3d.git
+git clone https://github.com/DeepLearnPhysics/spine.git
```

Comment:
François made a tag version for MR6 we could use
`git clone -b v0.1.0 https://github.com/DeepLearnPhysics/spine.git`

Reply:
Great suggestion!
```suggestion
git clone -b v0.1.0 https://github.com/DeepLearnPhysics/spine.git
```